client/my-sites/site-settings/form-general.jsx

```diff
@@ -707,6 +710,7 @@ const connectComponent = connect( ( state ) => {
 		siteDomains: getDomainsBySiteId( state, siteId ),
 		isWPForTeamsSite: isSiteWPForTeams( state, siteId ),
 		isP2HubSite: isSiteP2Hub( state, siteId ),
+		customizerUrl: getCustomizerUrl( state, siteId ),
```

Comment:
On Simple sites, we previously were redirecting users to `/customize/identify/:site` so the "Site Identity" section  (which contains the Footer Credit option) is automatically open.

We're now redirecting to `/customize/:site`, so the the "Site Identity" section is no longer open. Can we keep the previous behavior?

It'd be great to also replicate the behavior on Atomic sites.

I think something like this should work:
```suggestion
		customizerUrl: getCustomizerUrl( state, siteId, 'identity' ),
```

Reply:
Good catch! I'll give that a go.